src/assets/icons/accountChange.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We can wrap SVG icon with <SvgIcon/> of MUI and create Icon component. (https://mui.com/components/icons/#svgicon)
In this case we need to scale for a 24x24px viewport svg Icon. 

With my experience it was easy to use SVG icon after import as ReactComponent.
For example
``` import {ReactComponent as SaddleIcon} from "../assest/saddle.svg" ```
After import like this we can give hight and width as Props on the Component (Like reward button)

In this case we need to give `currentColor` on the svg icon so it could have the wrapper's color.

For example:

```<svg width="52" height="52" viewBox="0 0 52 52" fill="currentColor" xmlns="http://www.w3.org/2000/svg">```
The advantage of this method is that we can use svg without make special component. But for the simple icon we often use, it might be good to wrap with <SvgIcon/>, I think.